eglot-jl.jl

```diff
@@ -30,8 +30,21 @@ project_path = something(Base.current_project(src_path), Base.load_path_expand(L
 empty!(LOAD_PATH)
 push!(LOAD_PATH, "@")
 
+using LanguageServer.JSON
+if get(ENV, "EGLOT_JL_TEST", "0") == "1"
+    msg = Dict("jsonrpc" => "2.0",
+               "id"      => 1,
+               "method"  => "exit",
+               "params"  => Dict()) |> JSON.json
+    input = IOBuffer("Content-Length: $(length(msg))\n\n$msg")
```

Comment:
I'm  surprised using PackageCompiler makes such a big difference when the type of input isn't consistent between the compilation run and the normal run.

Reply:
Yes, I would say this is because most of the compilation/inference time is spent in sub-calls which work on parsed requests that always have the same type.

But this is only an intuition and I don't really know for sure. I'm just glad such an easy solution works so well, because I'm not sure how requests could be sent to the server via `stdin` while it is being package-compiled.